FHEM/00_SIGNALduino.pm

```diff
@@ -2261,14 +2274,23 @@ sub SIGNALduino_Parse_MU($$$$@)
 			my $floatRegex ="";
 			my $signalRegex;
 			my $regex;
+			my $ProtocListClock;
 			
 			if (($pstr=SIGNALduino_PatternExists($hash,\@{$ProtocolListSIGNALduino{$id}{one}},\%patternList,\$rawData)) eq -1)
 			{
-				SIGNALduino_Log3 $name, 5, "$name: one pattern for MU Protocol id $id not found, aborting";
+				Log3 $name, 5, "$name: one pattern for MU Protocol id $id not found, aborting" if ($dummy);
 				next;
 			}
 			Debug "Found matched one" if ($debug);
-
+			if ($clocksource eq "one")		# clocksource one, dann die clock aus one holen
+			{
+				$msgclock = $msg_parts{pattern}{substr($pstr, $ProtocolListSIGNALduino{$id}{clockpos}[1], 1)};
```

Comment:
wenn in $ProtocolListSIGNALduino{$id}{clockpos}[1] eine Position steht, die es nicht gibt, dann gibt es hier mind. eine Warnung.

Reply:
Syntaxprüfungen und Plausiblitätsprüfungen während der Laufzeit finde ich nicht so gut. Kann so was nicht durch Prüfungen und Tests in Travis erfolgen?

Reply:
Da User immer Mal wieder angewiesen werden etwas in der Protokoll Definition zu ändern würde ich das beim Laden der Protokoll Liste erledigen wollen und mich nicht auf Travis verlassen.